packages/core/Plugin.ts

```diff
@@ -9,6 +10,8 @@ export default abstract class Plugin {
   install(_pluginManager: PluginManager): void {}
 
   configure(_pluginManager: PluginManager): void {}
+
+  configuration: AnyConfigurationSchemaType | undefined = undefined
```

Comment:
probably want to rename this to `configurationSchema`, cause it is a schema, not a config with data in it

Reply:
You got it! 👍  Will go ahead and make that change.